johnsonandjohnson/Bodiless-JS
# chore(examples, test-site): Update Bodiless start to run site from sites folder

## Changes


## Test Instructions

* Run npm start from root
* Run npm start from starter site folder
* Deploy to platform.sh and confirm all deploys as expected.

## Related Issues

Fixes #1219
